docs/ai/quickstarts/structured-output.md

```diff
@@ -0,0 +1,115 @@
+
+## Prerequisites
+
+- [.NET 8 or a later version](https://dotnet.microsoft.com/download)
+- [Visual Studio Code](https://code.visualstudio.com/) (optional)
+
+## Configure the AI service
+
+To provision an Azure OpenAI service and model using the Azure portal, complete the steps in the [Create and deploy an Azure OpenAI Service resource](/azure/ai-services/openai/how-to/create-resource?pivots=web-portal) article. In the "Deploy a model" step, select the `gpt-4o` model.
+
+## Create the chat app
+
+Complete the following steps to create a console app that connects to the `gpt-4o` AI model.
+
+1. In a terminal window, navigate to the directory where you want to create your app, and create a new console app with the `dotnet new` command:
+
+    ```dotnetcli
+    dotnet new console -o SOChat
+    ```
+
+1. Navigate to the `SOChat` directory, and add the necessary packages to your app:
+
+    ```dotnetcli
+    dotnet add package Azure.AI.OpenAI
+    dotnet add package Azure.Identity
+    dotnet add package Microsoft.Extensions.AI --prerelease
+    dotnet add package Microsoft.Extensions.AI.OpenAI --prerelease
+    dotnet add package Microsoft.Extensions.Configuration
+    dotnet add package Microsoft.Extensions.Configuration.UserSecrets
+    ```
+
+1. Run the following commands to add [app secrets](/aspnet/core/security/app-secrets) for your Azure OpenAI endpoint, model name, and tenant ID:
+
+    ```bash
+    dotnet user-secrets init
+    dotnet user-secrets set AZURE_OPENAI_ENDPOINT <your-azure-openai-endpoint>
+    dotnet user-secrets set AZURE_OPENAI_GPT_NAME gpt-4o
+    dotnet user-secrets set AZURE_TENANT_ID <your-tenant-id>
+    ```
+
+   > [!NOTE]
+   > Depending on your environment, the tenant ID might not be needed. In that case, remove it from the code that instantiates the <xref:Azure.Identity.DefaultAzureCredential>.
+
+1. Open the new app in your editor of choice.
+
+## Add the code
+
+1. Define the enumeration that describes the different sentiments.
+
+   :::code language="csharp" source="./snippets/structured-output/Program.cs" id="SentimentEnum":::
+
+1. Create the <xref:Microsoft.Extensions.AI.IChatClient> that will communicate with the model.
+
+   :::code language="csharp" source="./snippets/structured-output/Program.cs" id="GetChatClient":::
+
+   > [!NOTE]
+   > <xref:Azure.Identity.DefaultAzureCredential> searches for authentication credentials from your environment or local tooling. You'll need to assign the `Azure AI Developer` role to the the tenant ID account or the account you used to sign in to Visual Studio or the Azure CLI. For more information, see [Authenticate to Azure AI services with .NET](../azure-ai-services-authentication.md).
```

Comment:
```suggestion
   > <xref:Azure.Identity.DefaultAzureCredential> searches for authentication credentials from your environment or local tooling. You'll need to assign the `Azure AI Developer` role to the account you used to sign in to Visual Studio or the Azure CLI. For more information, see [Authenticate to Azure AI services with .NET](../azure-ai-services-authentication.md).
```

My understanding is the tenant ID helps DAC to locate the account if the account is associated with multiple tenants, I don't think there is such thing as "tenant ID account".